tests/testgenerator.py

```diff
@@ -325,19 +295,16 @@ def test_pointer_depth(self):
     def test_reference_member(self):
         result = Namespace(name = "Capdpa", children = [
             Class(name = "With_Reference", children = [
-                Member(name = "R", ctype = Type_Reference(name = Identifier(["Int"]), reference = True))])]).AdaSpecification()
-        self.check(result[0].Text(), self.load("test_capdpa.txt"))
-        self.check(result[1].Text(), self.load("test_reference_member.txt"))
+                Member(name = "R", ctype = Type_Reference(name = Identifier(["Capdpa", "Int"]), reference = True))])]).AdaSpecification()
+        self.check(result[0].Text(), self.load("generator/test_reference_member.txt"))
 
     def test_spec_file_name(self):
         result = Namespace(name = "Capdpa", children = [
             Class(name = "Child1"),
             Namespace(name = "Child2", children = [
                 Class(name = "Grandchild")])]).AdaSpecification()
         self.check(result[0].FileName(), "capdpa.ads")
-        self.check(result[1].FileName(), "capdpa-child1.ads")
-        self.check(result[2].FileName(), "capdpa-child2.ads")
-        self.check(result[3].FileName(), "capdpa-child2-grandchild.ads")
+        self.check(result[1].FileName(), "capdpa-child2.ads")
```

Comment:
Maybe we should extend this test again to create a grand child package again? The initial goal of this test was to test file name generation for different depths of child packages.